src/graphql/get-resolution.query.ts

```diff
@@ -0,0 +1,17 @@
+import { gql } from "graphql-request";
+
+export const getResolutionQuery = gql`
+  query GetResolution($id: String!) {
+    resolution(id: $id) {
+      id
+      title
+      content
+      isNegative
+      typeId
+      yesVotesTotal
+      approveTimestamp
+      createTimestamp
+      updateTimestamp
```

Comment:
We will prolly add this to a fragment to avoid duplication, but let's keep it like this for now